cscs-checks/apps/cpmd/cpmd_check.py

```diff
@@ -20,13 +20,12 @@ def __init__(self, **kwargs):
         self.num_tasks_per_node = 1
         self.use_multithreading = True
 
-        found_energy = sn.extractsingle(
+        energy = sn.extractsingle(
             r'CLASSICAL ENERGY\s+-(?P<result>\S+)',
             'stdout.txt', 'result', float)
-
-        self.sanity_patterns = sn.assert_reference(
-            found_energy, 25.812675, -1e-2, 1e-2)
-
+        energy_reference = 25.81
+        energy_diff = sn.abs(energy-energy_reference)
```

Comment:
Please leave whitespace around the `-` operator. Check out the [coding style](https://github.com/eth-cscs/reframe/wiki/Coding-Style-Guide#formatting-operators).